src/FeaturedComponents.jsx

```diff
@@ -62,6 +62,12 @@ return (
           src="${REPL_ACCOUNT}/widget/ComponentCard"
           props={{ src: "devgovgigs.near/widget/Ideas" }}
         />
+      </Item>
+            <Item>
+        <Widget
+          src="${REPL_ACCOUNT}/widget/ComponentCard"
+          props={{ src: "nomination.ndctools.near/widget/NDC.Nomination.Page" }}
```

Comment:
@codingshot do you have a testnet version of this component? If so, please commit an update to the replacements.*.json files as mentioned in [this PR](https://github.com/near/near-discovery-components/pull/167#discussion_r1293855988) which you can use as a reference.